doc/source/user_guide/io.rst

```diff
@@ -3287,6 +3287,45 @@ output (as shown below for demonstration) for easier parse into ``DataFrame``:
    df = pd.read_xml(xml, stylesheet=xsl)
    df
 
+For very large XML files that can range in hundreds of megabytes to gigabytes, :func:`pandas.read_xml`
```

Comment:
Is this entire xml section in the user guide linked to the `read_xml` docstring? If not it would be good to link them under the `Notes` of a docstring

Reply:
Good idea! Done.